src/panel/category/CategoryPanel.jsx

```diff
@@ -0,0 +1,227 @@
+/* globals _ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import Panel from 'src/components/ui/Panel';
+import PoiCategoryItems from './PoiCategoryItems';
+import debounce from 'src/libs/debounce';
+import IdunnPoi from 'src/adapters/poi/idunn_poi';
+import nconf from '@qwant/nconf-getter';
+import Telemetry from 'src/libs/telemetry';
+import CategoryService from 'src/adapters/category_service';
+import SearchInput from 'src/ui_components/search_input';
+import layouts from 'src/panel/layouts.js';
+import { sources } from 'config/constants.yml';
+
+const MAX_PLACES = Number(nconf.get().category.maxPlaces);
+
+export default class CategoryPanel extends React.Component {
+  static propTypes = {
+    categoryName: PropTypes.string.isRequired,
+    query: PropTypes.string,
+    bbox: PropTypes.string,
```

Comment:
`categoryName` is actually NOT required. It should be possible to use `query` without any category name.  
And a URL such as `/places/?q=myQuery` should lead to a list of results. This currently raises a JS error. (A test case is probably missing about this :disappointed: )